deployment/terraform/variables.tf

```diff
@@ -1,48 +1,50 @@
-variable "stack_name" {
-  default = "GeoTrellis Site"
-  type = "string"
-  description = "Disambiguation prefix for the EMR/ECS stack"
+variable "environment" {
+  default = "Production"
 }
 
-variable "service_image" {
-  type = "string"
-  description = "Geotrellis service Docker image"
+variable "remote_state_bucket" {
+  type        = "string"
+  description = "Core infrastructure config bucket"
 }
 
-variable "static_image" {
-  type = "string"
-  description = "Static service Docker image"
+variable "aws_account_id" {
+  default     = "896538046175"
+  description = "Geotrellis Site account ID"
 }
 
-variable "ec2_key" {
-  default = "geotrellis-cluster"
-  type = "string"
-  description = "EC2 key for EMR and ECS machines"
+variable "aws_region" {
+  default = "us-east-1"
 }
 
-variable "subnet_id" {
-  default = "subnet-c5fefdb1"
-  type = "string"
-  description = "Subnet ID shared by EMR and ECS"
+variable "git_commit" {
+  type        = "string"
+  description = "Geotrellis Service and Static Image"
 }
-variable "desired_instance_count" {
-  default = 1
-  description = "Number benchmark instances to provision"
+
+variable "cdn_price_class" {
+  default = "PriceClass_200"
+}
+
+variable "ssl_certificate_arn" {
+  default = "arn:aws:acm:us-east-1:896538046175:certificate/a416c2af-00dd-4afd-8c71-dd32edefa839"
+}
+
+variable "website_ecs_desired_count" {
+  default = "2"
 }
 
-# TODO: make this a dynamic lookup
-variable "aws_ecs_ami" {
-  default = "ami-6bb2d67c"
+variable "website_ecs_min_count" {
+  default = "2"
 }
 
-variable "ecs_instance_type" {
-  default = "t2.medium"
+variable "website_ecs_max_count" {
+  default = "2"
 }
 
-variable "ecs_service_role" {
-  default = "arn:aws:iam::896538046175:role/ecs_service_role"
+variable "website_ecs_deployment_min_percent" {
+  default = "0"
```

Comment:
This we should default to 100 because we want to avoid downtime when things get deployed.